### PR TITLE
[GHA] Return Smart CI condition for Python unit tests

### PR DESCRIPTION
**Pull request by akladiev**
Checking if CVS-127101 still reproduces (it doesn't: https://github.com/openvinotoolkit/openvino/actions/runs/9550495926/job/26323459000?pr=25079, see passing MO unit tests)

### CODE REVIEW

File: .github/workflows/job_python_unit_tests.yml [hunk 1]
```diff
@@ -121,7 +121,7 @@ jobs:
       #
 
       - name: Python API Tests
-        # if: ${{ fromJSON(inputs.affected-components).Python_API.test && runner.arch != 'ARM64' }} # Ticket: 126380, 127101
+        if: ${{ fromJSON(inputs.affected-components).Python_API.test && runner.arch != 'ARM64' }} # Ticket: 126380
```
Comment (rkazants) [inline .github/workflows/job_python_unit_tests.yml:124]: don't we want to run it on ARM?

Comment (akladiev) [reply .github/workflows/job_python_unit_tests.yml]: We do, but there was an issue on ARM: 126380

**Comment by rkazants** [comment]
Let us disable concrete tests on ARM instead. Will it help? Now we exclude running all tests on ARM.

File: .github/workflows/job_python_unit_tests.yml (hunk 1 above)
Comment (akladiev) [reply .github/workflows/job_python_unit_tests.yml]: Ok, they doesn't reproduce anyway

**Review by ilya-lavrenov — APPROVED** [review]

**Pull request merged**